storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/PutOperationTest.java

```diff
@@ -0,0 +1,340 @@
+        changesTestSubscriber.assertNoErrors();
+        changesTestSubscriber.assertValue(Changes.newInstance(TestItem.CONTENT_URI));
+    }
+
+    @Test
+    public void updateContentValuesExecuteAsBlocking() {
+        TestSubscriber<Changes> changesTestSubscriber = new TestSubscriber<Changes>();
+
+        storIOContentResolver
+                .observeChangesOfUri(TestItem.CONTENT_URI)
+                .take(2)
+                .subscribe(changesTestSubscriber);
+
+        Uri insertedUri = contentResolver.insert(TestItem.CONTENT_URI, TestItem.create(null, "value").toContentValues());
+
+        TestItem testItem = TestItem.create(ContentUris.parseId(insertedUri), "value2");
+
+        PutResult updateResult = storIOContentResolver
+                .put()
+                .contentValues(testItem.toContentValues())
+                .withPutResolver(testItemContentValuesPutResolver)
+                .prepare()
+                .executeAsBlocking();
+
+        assertThat(updateResult.wasUpdated()).isTrue();
+
+        Cursor cursor = contentResolver.query(TestItem.CONTENT_URI, null, null, null, null);
+
+        Assertions.assertThat(cursor).hasCount(1);
+
+        cursor.moveToFirst();
+
+        assertThat(testItem).isEqualTo(TestItem.fromCursor(cursor));
+
+        changesTestSubscriber.awaitTerminalEvent(60, SECONDS);
+        changesTestSubscriber.assertNoErrors();
+        changesTestSubscriber.assertValues(Changes.newInstance(TestItem.CONTENT_URI), Changes.newInstance(TestItem.CONTENT_URI));
+    }
+
+    @Test
+    public void updateContentValuesAsObservable() {
+        TestSubscriber<Changes> changesTestSubscriber = new TestSubscriber<Changes>();
+
+        storIOContentResolver
+                .observeChangesOfUri(TestItem.CONTENT_URI)
+                .take(2)
+                .subscribe(changesTestSubscriber);
+
+        Uri insertedUri = contentResolver.insert(TestItem.CONTENT_URI, TestItem.create(null, "value").toContentValues());
+
+        TestItem testItem = TestItem.create(ContentUris.parseId(insertedUri), "value2");
+
+        PutResult updateResult = storIOContentResolver
+                .put()
+                .contentValues(testItem.toContentValues())
+                .withPutResolver(testItemContentValuesPutResolver)
+                .prepare()
+                .createObservable()
+                .toBlocking()
+                .first();
+
+        assertThat(updateResult.wasUpdated()).isTrue();
+
+        Cursor cursor = contentResolver.query(TestItem.CONTENT_URI, null, null, null, null);
+
+        Assertions.assertThat(cursor).hasCount(1);
+
+        cursor.moveToFirst();
+
+        assertThat(testItem).isEqualTo(TestItem.fromCursor(cursor));
+
+        changesTestSubscriber.awaitTerminalEvent(60, SECONDS);
+        changesTestSubscriber.assertNoErrors();
+        changesTestSubscriber.assertValues(Changes.newInstance(TestItem.CONTENT_URI), Changes.newInstance(TestItem.CONTENT_URI));
```

Comment:
Can we move these calls into the one method? Just to minimize copypaste and prevent a lot of changes if we'll decide to use 30 seconds as wait interval, for example?